pandas/tests/series/test_operators.py

```diff
@@ -976,16 +1012,6 @@ def run_ops(ops, get_ser, test_ser):
         # ## timedelta64 ###
```

Comment:
is this initialization code then used below?

Reply:
Yes.